source/internals.js

```diff
@@ -0,0 +1,242 @@
+const { spawn } = require('child_process');
```

Comment:
This file became rather big, I would rather create an 'internals' folder with the tasksList, the ask* and the gets into its own files.

Reply:
I would agree with that as well - it's pretty cluttered right now, but again I wanted to do something more minimal, like a Proposal Request more, first, since you're receptive I'll take your advice and re-shape this